Bin/Snapshot.php

```diff
@@ -273,16 +304,53 @@ function () use ($tags, $newTag, $repositoryRoot, &$changelog) {
                             $title = $newTag;
                         }
 
-                        $changelog .=
-                            '# ' . $title . "\n\n" .
-                            Console\Processus::execute(
-                                'git --git-dir=' . $repositoryRoot . '/.git ' .
-                                    'log ' .
-                                        '--first-parent ' .
-                                        '--pretty="format:  * %s (%aN, %aI)" ' .
-                                        $fromStep . '...' . $toStep,
-                                false
-                            ) . "\n\n";
+                        $changelog .= '# ' . $title . "\n\n";
+
+                        $commits = Console\Processus::execute(
```

Comment:
```
- $commits = Console\Processus::execute(
+ $commits = explode(PHP_EOL, Console\Processus::execute(
```

This way you can remove the `$collection` variable, which has not a very well-defined meaning regarding its name.